Core.ApplicationServices/SystemUsage/ItSystemUsageService.cs

```diff
@@ -334,6 +334,32 @@ public Maybe<OperationError> RemoveRelevantUnit(int id, Guid unitUuid)
             }).MatchFailure();
         }
 
+        public Maybe<OperationError> AddPersonalDataOption(int id, GDPRPersonalDataOption option)
+        {
+            return Modify(id, system =>
+            {
+                return system.AddPersonalData(option)
+                    .Match
+                    (
+                        error => error,
+                        () => Result<ItSystemUsage, OperationError>.Success(system)
+                    );
+            }).MatchFailure();
+        }
+
+        public Maybe<OperationError> RemovePersonalDataOption(int id, GDPRPersonalDataOption option)
+        {
+            return Modify(id, system =>
+            {
```

Comment:
You are leaking options because
- You have no explicit mapping of ownership so the default is to allow null as itsystemusage
![image](https://user-images.githubusercontent.com/13116889/207253518-465d34b9-be7f-4932-8ea7-8ad7d34e26a5.png)

Leaks in the cases where you:
- Remove one (this instance)
- Remove all (when person data is disabled)

Reply:
remember to test the remove cases in the integration test